lte/gateway/python/integ_tests/s1aptests/util/traffic_util.py

```diff
@@ -135,15 +135,15 @@ def exec_command(self, command):
 
     def update_dl_route(self, ue_ip_block):
         """Update downlink route in TRF server"""
-        ret_code = self.exec_command(
+        ret_code_ipv4 = self.exec_command(
             "sudo ip route flush via 192.168.129.1 && sudo ip route "
             "replace " + ue_ip_block + " via 192.168.129.1 dev eth2",
         )
-        ret_code = self.exec_command(
+        ret_code_ipv6 = self.exec_command(
             "sudo ip -6 route flush via " + self.agw_ipv6 + " && sudo ip -6 route "
             "replace " + self.ue_ipv6_block + " via " + self.agw_ipv6 + " dev eth3",
```

Comment:
Maybe out of scope, but where the `ue_ip_block` and the `ue_ipv6_block` are coming from is inconsistent, same with the hard-coded ipv4 and the `agw_ipv6`.

Reply:
That is correct. Both, `self.agw_ipv6` and `self.ue_ipv6_block`, are only used here. @pruthvihebbani?

Reply:
Ipv6 addresses are hardcoded for now. I will create a new PR to handle this similar to ipv4.